shared/params/io_config.go

```diff
@@ -0,0 +1,18 @@
+package params
+
+import "os"
+
+// IoConfig defines the shared io parameters.
+type IoConfig struct {
+	FilePermission os.FileMode
+}
+
+var defaultIoConfig = &IoConfig{
+	FilePermission: 0600, //-rw------- Read and Write permissions for user
```

Comment:
Can we name this to be more clear? Something like `ReadWritePermissions`

Reply:
renamed